app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeTicketUi.kt

```diff
@@ -140,23 +139,26 @@ fun QrCodeTicketUi(
               }
         }
 
+    val componentStyle =
+        EventComponentsStyle(
+            MaterialTheme.colorScheme.onSurface,
+            MaterialTheme.colorScheme.onSurfaceVariant,
+            MaterialTheme.colorScheme.onSurface,
+        )
+
     if (qrScanUiState.value.tabState == ScanTicketQrViewModel.Tab.MyEvent) {
       val imageHeight = 191.dp
 
       val uiState = viewModel.uiState.collectAsStateWithLifecycle().value
 
-      val componentStyle =
-          EventComponentsStyle(
-              MaterialTheme.colorScheme.onSurface,
-              MaterialTheme.colorScheme.onSurfaceVariant,
-              MaterialTheme.colorScheme.onSurface,
-          )
-
       val (lazyEventDetails) = createRefs()
 
-      // TODO uncomment when image are implemented
-      // val imagePainter: Painter = rememberImagePainter(eventUiState.eventPhoto)
-      val imagePainter: Painter = rememberImagePainter(R.drawable.placeholderbig)
+      val imagePainter =
+          if (qrScanUiState.value.eventUiState.eventPhoto == "") {
```

Comment:
this is causing a bug, just tried when running preview, please try and fix. I would suggest to do "
val imagePainter =
           if (qrScanUiState.value.eventUiState.eventPhoto == "") {
rememberImagePainter(eventUiState.eventPhoto) else {
rememberAsyncImagePainter(qrScanUiState.value.eventUiState.eventPhoto)} and see if that works